### PR TITLE
Renames and relores loyalty implants to mindshield implants.

### PR DESCRIPTION
**Pull request by TechnoAlchemisto**
For @KorPhaeron and @Shapsy 
:cl: Papa Bones
tweak: Loyalty implants have been refluffed to mindshield implants, they are mechanically the same.
/:cl:
Testimonials: 
FUCK

Don't forget to like comment and subscribe.

### CODE REVIEW

File: code/modules/research/designs/weapon_designs.dm [hunk 1]
```diff
@@ -14,12 +14,12 @@
 
 /datum/design/pin_loyalty
 	name = "loyalty firing pin"
```
Comment (Cheridan) [inline code/modules/research/designs/weapon_designs.dm]: didn't change the name :V

**Comment by Cheridan** [comment]
I guess this fixes the problem of people thinking that implants do anything against wizards/traitors and such, but I still like the roleplay that loyalty implants bring (to the few people who roleplay loyalty to NT after an implant, at least).

**Comment by Steelpoint** [comment]
As I said before this is a positive change as Cheridan explains well.

Actual Loyalty to NT is something that a character holds, either via training, actual loyalty or being bought by them. Thus you would assume anyone in a high ranking command or a security position hold a high loyatly to NT from the get go, hence why the Captain and Security (barring that loose cannon Detective) are loyal enough to not be a round start antagonist.

File: code/modules/projectiles/pins.dm [hunk 2]
```diff
@@ -100,11 +100,11 @@
 			return 1
 	return 0
 
-/obj/item/device/firing_pin/implant/loyalty
-	name = "loyalty firing pin"
-	desc = "This Security firing pin authorizes the weapon for only loyalty-implanted users."
+/obj/item/device/firing_pin/implant/mindshield
+	name = "mindsheild firing pin"
```
Comment (coiax) [inline code/modules/projectiles/pins.dm]: mindshield

File: code/modules/mob/living/carbon/human/human.dm [hunk 3]
```diff
@@ -701,7 +701,7 @@
 	if(dna && dna.species.id && dna.species.id != "human")
```
Comment (coiax) [inline code/modules/mob/living/carbon/human/human.dm]: should be changed to ishumanbasic()

File: code/game/objects/items/weapons/implants/implant_loyality.dm [hunk 4]
```diff
@@ -1,65 +1,65 @@
...
-			target.visible_message("<span class='warning'>[src] was destroyed in the process!</span>", "<span class='notice'>You feel a surge of loyalty towards Nanotrasen.</span>")
+			target.visible_message("<span class='warning'>[src] was destroyed in the process!</span>", "<span class='notice'>You feel a sense of peace and security. You are now protected from brainwashing.</span>")
 			removed(target, 1)
 			qdel(src)
 			return -1
 		if(target.mind in ticker.mode.revolutionaries)
 			ticker.mode.remove_revolutionary(target.mind)
 		if((target.mind in ticker.mode.cult) || (target.mind in ticker.mode.blue_deity_prophets|ticker.mode.red_deity_prophets|ticker.mode.red_deity_followers|ticker.mode.blue_deity_followers))
-			target << "<span class='warning'>You feel the corporate tendrils of Nanotrasen try to invade your mind!</span>"
+			target << "<span class='warning'>You feel something try to heal your mind, but you resist it!</span>"
```
Comment (coiax) [inline code/game/objects/items/weapons/implants/implant_loyality.dm]: I feel like this is a bit weird from the perspective of the revhead. Why not "You feel something interfering with your mental conditioning" or something like that. I don't think the brainwashed(may not be canon) revheads consider themselves damaged.

**Comment by Robustin** [comment]
Such a rare issue to require a change that will undoubtedly confuse other people who have been told/trained to use loyalty implants.

I can't remember the last time someone tried to loyalty implant a traitor/ling/whatever.

File: code/modules/mob/living/carbon/human/human.dm (hunk 3 above)
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/human.dm]: how2basic()

File: code/game/gamemodes/handofgod/_handofgod.dm [hunk 5]
```diff
@@ -35,7 +35,7 @@ var/global/list/global_handofgod_structuretypes = list()
 	world << "<B>Two cults are onboard the station, seeking to overthrow the other, and anyone who stands in their way.</B>"
 	world << "<B>Followers</B> - Complete your deity's objectives. Convert crewmembers to your cause by using your deity's nexus. Remember - there is no you, there is only the cult."
 	world << "<B>Prophets</B> - Command your cult by the will of your deity.  You are a high-value target, so be careful!"
-	world << "<B>Personnel</B> - Do not let any cult succeed in its mission. Loyalty implants and holy water will revert them to neutral, hopefully nonviolent crew."
+	world << "<B>Personnel</B> - Do not let any cult succeed in its mission. Mindshield implants and holy water will revert them to neutral, hopefully nonviolent crew."
```
Comment (ChangelingRain) [inline code/game/gamemodes/handofgod/_handofgod.dm]: It's just holy water but this message basically never shows up

Comment (coiax) [inline code/game/gamemodes/handofgod/_handofgod.dm]: also RIP gamemode code when I'm done

**Comment by PKPenguin321** [comment]
I like the idea of creating mindless corporate slaves more than a brain condom

**Comment by TechnoAlchemisto** [comment]
Ya except they don't make you loyal, I mean I love roleplay and shit but having an assistant yell "hail nanotrasen" before going back to smashing windows and breaking shit without roleplaying any personality change is hardly roleplay.

**Pull request merged**